src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -368,6 +368,7 @@ class Meta:
             "processobjectaard",
             "startdatum_bewaartermijn",
             "processobject",
+            "communicatiekanaal_naam",
```

Comment:
Can you move this line to just below `communicatiekanaal`? That way they show up after eachother in the API schema

Reply:
Moved